test/repl.jl

```diff
@@ -348,7 +348,7 @@ begin
     LineEdit.accept_result(s, histp)
     @test LineEdit.mode(s) == repl_mode
     @test buffercontents(LineEdit.buffer(s)) == "shell"
-    @test position(LineEdit.buffer(s)) == 4
+    @test position(LineEdit.buffer(s)) == 2
```

Comment:
I'll take a look at this. FWIW, you can debug LineEdit using Gallium by loading a separate copy of LineEdit outside the Base module and debugging that.